Application/EdFi.Ods.SandboxAdmin/wwwroot/js/client.js

```diff
@@ -122,29 +133,94 @@ var ResetDialog = function () {
 var AddApplicationDialog = function () {
     var self = this;
 
-    self.applicationName = ko.observable('');
+    self.fieldsTouched = ko.observable(false);
+    self.buttonText = ko.observable("");
+    self.sandboxName = ko.observable(null).extend({
+        required: {
+            message: 'The Sandbox Name field is required.',
+            onlyIf: function () { return self.fieldsTouched(); }
+        },
+        pattern: {
+        message: 'Please enter only letters',
+        params: /^[A-Za-z]+$/
+    }
+    });
+
     self.useSampleData = ko.observable(false);
     self.applicationList = ko.observableArray();
-    self.selectedApplication = ko.observable();
-    self.canAdd = ko.computed(function () {
-        return self.applicationName().length > 0;
+
+    self.selectedApplication = ko.observable(null).extend({
+        required: {
+            message: 'Please select a application.',
```

Comment:
Please change 'a' to 'an' here and on L161.

Reply:
Fixed It . Thanks